node/src/crypto/asymmetric_key.rs

```diff
@@ -493,12 +511,12 @@ MCowBQYDK2VwAyEAGb9ECWmEzf6FQbrBZ9w7lshQhqowtrbLDFw4rXAxZuE=
             let ed25519_secret_key = SecretKey::random_ed25519(&mut rng);
             let public_key = PublicKey::from(&ed25519_secret_key);
             let data = b"data";
-            let signature = sign(data, &ed25519_secret_key, &public_key, &mut rng);
+            let signature = sign(data, &ed25519_secret_key, &public_key);
             bytesrepr::test_serialization_roundtrip(&signature);
         }
     }
 
-    mod secp256k1 {
+    mod test_secp256k1 {
```

Comment:
Should remove `test_` prefix.

Reply:
Sadly, we can't because this collides with a library with the same name.

Reply:
This was colliding with the crate with the same name, so I changed it to have this prefix.  I also changed `ed25519` for symmetry.